add support ffmpeg 6.1

im having issues getting 6.1 to be used, im getting this error:
```
error[E0587]: type has conflicting packed and align representation hints
 --> C:\Projects\Rust\rs_gtproj\target\debug\build\rusty_ffmpeg-1acc2c7ce02b39bd\out/binding.rs:3:81514
  |
3 | ... # [repr (C , packed (8))] # [repr (align (8))] # [derive (Copy , Clone)] pub union __mingw_ldbl_type_t { pub x : u128 , pub lh : __mingw_ldbl_type_t__bindgen_ty_1 , } # [repr (C... 
  |         
 ```
any suggestions? im using cargo-vcpkg on windows with the line `x86_64-pc-windows-msvc = { triplet = "x64-windows-static-md", dependencies = ["ffmpeg"] }` for the `package.metadata.vcpkg.target` bit in the toml

--- a/build.rs
+++ b/build.rs
@@ -229,6 +229,8 @@ fn generate_bindings(ffmpeg_include_dir: &Path, headers: &[PathBuf]) -> Bindings
                     .parse_callbacks(Box::new(filter_callback))
                     // Add clang path, for `#include` header finding in bindgen process.
                     .clang_arg(format!("-I{}", ffmpeg_include_dir))
+                    // Workaround: https://github.com/rust-lang/rust-bindgen/issues/2159
+                    .blocklist_type("__mingw_ldbl_type_t")
                     // Stop bindgen from prefixing enums
                     .prepend_enum_name(false)
             },

